addons/packages/knative-serving/1.5.0/package.yaml

```
@@ -0,0 +1,132 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: knative-serving.community.tanzu.vmware.com.1.5.0
+spec:
+  refName: knative-serving.community.tanzu.vmware.com
+  version: 1.5.0
+  releaseNotes: "knative-serving 1.5.0 https://github.com/knative/serving/releases/tag/v1.5.0"
+  licenses:
+    - "Apache 2.0"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects.registry.vmware.com/tce/knative-serving@sha256:648b8eb811806513e9681912899be3bdd086b8b7d99d0b7d52dbf44024ee858f
```

```suggestion
            image: projects.registry.vmware.com/tce/knative-serving@sha256:01dd43ed368cb42985204916aacd64c9ef0b159b3b7436d4de8076d1134bf4b6
```
@nader-ziada I pushed the package to our OCI registry. Please accept this suggestion with the new image URL and I can get this merged!